src/include/Context.h

```diff
@@ -473,18 +473,16 @@ class C_GatherBuilderBase
 typedef C_GatherBase<Context, Context> C_Gather;
 typedef C_GatherBuilderBase<Context, C_Gather > C_GatherBuilder;
 
+template<typename T>
```

Comment:
please note, we also have `LambdaContext` added by @athanatos . shall we just drop one of them?

Reply:
Thanks for pointing that out. Yes, I'll do this cleanup later today.

Reply:
done